### DISCUSSION TITLE
More shorcuts

### DISCUSSION THREAD

**Discussion by ehernandezvilla** [dailydotdev/daily#1497] [Feature Requests]
Hey guys, it would be great if you can add the option to add more shorcuts. 
![image](https://github.com/user-attachments/assets/df1d1e8e-b3d4-45ba-932b-31cfcea7cb79)

**Comment by idoshamun** [comment]
Hi @ehernandezvilla,

Thanks for the suggestion. Closing as it's dup of https://github.com/dailydotdev/daily/discussions/1387
